src/Output/OutputFactory.php

```diff
@@ -28,17 +28,17 @@ private function buildBestOutput(bool $forceTty = null, bool $forceNoTty = null)
         }
 
         if ($forceNoTty) {
-            return new Simple();
+            return new PhpunitLike();
```

Comment:
As the Simple class is still in the code, and because we can not change the display, it means we have dead code in this project :/
May be we could add an option for the display, with default value to `PhpUnitLike`